cmsplugin_articles_ai/management/commands/publish_test_articles.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
The management command's name should be prefixed with some sort of namespace to prevent it from clashing with other projects.